src/pages/trainer/training/edit/components/preview/preview.tsx

```diff
@@ -3,11 +3,19 @@ import { MarkDownViewerComponent } from '../../../../../../common/components/mar
 
 interface Props {
   content: string;
+  scrollSourceLine?: number;
+  onScrollSourceLine?: (sourceLine) => any;
+  className?: string;
 }
 
-export const PreviewComponent: React.StatelessComponent<Props> = ({ content }) => {
+export const PreviewComponent: React.StatelessComponent<Props> = (props) => {
```

Comment:
I think this component is not necessary, isn't it?

Reply:
Not much needed, just a logic wrapper. Shall I remove it?

Reply:
**Done**. Removed component. [Link to commit](https://github.com/Lemoncode/LeanMood/pull/168/commits/a1a843d66184e120ac7db8a8eeec065c1e568ef4).